modules/components/style.jsx

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
+var hyphenateStyleName = require('react/lib/hyphenateStyleName');
+var reduce = require('lodash.reduce');
```

Comment:
Per discussion with @alexlande -- can you change to native `ARRAY.reduce`?

@alexlande @rgerstenberger -- OR, feel free to ticket out and do globally...